### DISCUSSION TITLE
Installazione non crea Tabelle

### DISCUSSION THREAD

**Discussion by PranzoStef** [wavelog/wavelog#1207] [General]
Buona sera. Tutto verde sul controllo iniziale. Durante l'installazione si ferma alla crazione delle Tabelle con la voce:

Creating database tables failed. Ajax crashed.
cosa posso fare?

**Comment by HB9HIL** [comment]
Hello

Can you tell us which operating system, MariaDB/MySQL Version, webserver and so on

**Reply by PranzoStef** [reply]
sorry. I didn't write this before.
System Debian 12
MySQL Version 8.4.3
Web Server Apache2
PHP 8.2.24
PC I5 4Gb Ram
(I'm IZ7ECX ex HB9DNI)

**Reply by HB9HIL** [reply]
The most important thing is that you have an empty database. I would recommend: Start from new and create in mysql only user, password and an empty database. Make sure you start from new, because when the installer was at the datatables there already happened something in the filesystem. So delete the folder and start again at 0

**Reply by PranzoStef** [reply]
I redid from scratch. It doesn't work.
The DB is empty.
After the error I noticed that something writes to the DB. See photo

![immagine](https://github.com/user-attachments/assets/da5d38c4-b3cc-4625-933b-e44407054d60)

**Reply by HB9HIL** [reply]
I tested with MySQL Version 8.4.3 and it worked seamless here. So I need some more information to try to reproduce this issue.

Can you please:
- make a screenshot of every tab in the installer please
- and paste the content of  `application/logs/installer_debug.log` here

**Reply by PranzoStef** [reply]
ok. 
I installed Cloudlog and it works perfectly.
attached is the installation
![immagine](https://github.com/user-attachments/assets/eab10792-c618-4db7-9bb6-b5cb4fbd4af3)
![immagine](https://github.com/user-attachments/assets/1ca76dee-9979-4aa7-ab53-c399164fd420)
![immagine](https://github.com/user-attachments/assets/416a19a8-fa82-4f8e-8d12-65f285839299)
![immagine](https://github.com/user-attachments/assets/aefdf3b7-2c51-4d67-8e79-cdff4ac69cfb)
![immagine](https://github.com/user-attachments/assets/a24b2405-6a16-49e4-8b12-f1006649cd91)
![immagine](https://github.com/user-attachments/assets/206492dc-be36-4b9d-921e-1e4554f3103e)

**Reply by HB9HIL** [reply]
@PranzoStef Okay I was able to reproduce this issue. The reason why it fails is the `'` in your City Name `Castrignano de' Greci` 

I will try to fix that. In the meantime please try again installing Wavelog and omit special chars in the City name. Just write `Castrignano de Greci`

**Reply by HB9HIL** [reply]
@PranzoStef Thanks for reporting this. I created an issue #1208  from this discussion and fixed that in #1209 . This doesn't help you now (simply avoid using single quotation marks `'` to successfully install wavelog) but it will help future users. Thanks 👍

**Reply by PranzoStef** [reply]
Thanks for the info. I am in Zurich now. As soon as I get back (next week), I will test. Thanks again.

**Reply by PranzoStef** [reply]
Funziona.
in the City field I wrote Castrignano de Greci

![immagine](https://github.com/user-attachments/assets/19a1fa20-6862-4282-bd9e-cbc5a2c6edca)


![immagine](https://github.com/user-attachments/assets/887b3701-ba47-4246-ab41-8871d09a674c)

Grazie. 73

**Reply by HB9HIL** [reply]
Great to hear :-)

Quick reminder: We need translators for italian 👍  Check out https://translate.wavelog.org
